draft-trammell-rains-protocol.md

```diff
@@ -365,12 +384,12 @@ A signature over an assertion contains the following information elements:
   the algorithm used.
 
 The signature protects all the information in an assertion as well as its own
-algorithm identifier, keyspace, keyphase, valid-since, and valid-until values;
-it does not protect other signatures on the assertion.
+algorithm identifier, keyspace identifier, key phase, valid-since, and
+valid-until values; it does not protect other signatures on the assertion.
 
-### Shards and Zones
+### Shards, Bloom Filters, and Zones {#shards-bloom-filters-zones}
```

Comment:
These should be two separate sections:

1. shards and whatever-we'll-call-bloomfilters (useful for negative proof only)
2. zones (complete grouping, useful for pos + neg, common in the tiny-zone case)